.github/workflows/00-pr-scanner.yaml

```
@@ -101,89 +75,168 @@ jobs:
-                  python-version: "3.9"
-                  cache: "pip"
-
-            - name: create env
-              run: ./create_env.sh
-              working-directory: system-tests
-
-            - name: Generate uuid
-              id: uuid
+                  GH_TOKEN: ${{ steps.app-token.outputs.token }}
+                  CORRELATION_ID: ${{ steps.dispatch-info.outputs.correlation_id }}
               run: |
```

actionlint (1.7.10)

[error] "github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)